,url,repository_url,labels_url,comments_url,events_url,html_url,id,node_id,number,title,user,labels,state,locked,assignee,assignees,milestone,comments,created_at,updated_at,closed_at,author_association,active_lock_reason,draft,pull_request,body,reactions,timeline_url,performed_via_github_app,state_reason
0,https://api.github.com/repos/pandas-dev/pandas/issues/48736,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48736/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48736/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48736/events,https://github.com/pandas-dev/pandas/pull/48736,1383887193,PR_kwDOAA0YD84_gLgi,48736, BUG: AttributeError: 'function' object has no attribute 'currentframe' ,"{'login': 'MarcoGorelli', 'id': 33491632, 'node_id': 'MDQ6VXNlcjMzNDkxNjMy', 'avatar_url': 'https://avatars.githubusercontent.com/u/33491632?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/MarcoGorelli', 'html_url': 'https://github.com/MarcoGorelli', 'followers_url': 'https://api.github.com/users/MarcoGorelli/followers', 'following_url': 'https://api.github.com/users/MarcoGorelli/following{/other_user}', 'gists_url': 'https://api.github.com/users/MarcoGorelli/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/MarcoGorelli/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/MarcoGorelli/subscriptions', 'organizations_url': 'https://api.github.com/users/MarcoGorelli/orgs', 'repos_url': 'https://api.github.com/users/MarcoGorelli/repos', 'events_url': 'https://api.github.com/users/MarcoGorelli/events{/privacy}', 'received_events_url': 'https://api.github.com/users/MarcoGorelli/received_events', 'type': 'User', 'site_admin': False}",[],open,False,,[],,0,2022-09-23T14:25:38Z,2022-09-23T14:32:55Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48736', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48736', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48736.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48736.patch', 'merged_at': None}","- [ ] closes #48733  (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

Note how `pylint` would've caught this:
```
$ pylint pandas/io/sql.py | grep 'inspect'
pandas/io/sql.py:1648:12: W0621: Redefining name 'inspect' from outer scope (line 15) (redefined-outer-name)
pandas/io/sql.py:1648:12: C0415: Import outside toplevel (sqlalchemy.inspect) (import-outside-toplevel)
pandas/io/sql.py:1663:48: E1101: Function 'inspect' has no 'currentframe' member (no-member)
pandas/io/sql.py:1760:8: W0621: Redefining name 'inspect' from outer scope (line 15) (redefined-outer-name)
pandas/io/sql.py:1760:8: C0415: Import outside toplevel (sqlalchemy.inspect) (import-outside-toplevel)
```

We should add that to CI","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48736/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48736/timeline,,
1,https://api.github.com/repos/pandas-dev/pandas/issues/48735,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48735/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48735/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48735/events,https://github.com/pandas-dev/pandas/pull/48735,1383845141,PR_kwDOAA0YD84_gCr7,48735,DEPR: positional arguments in favor of keyword arguments,"{'login': 'ambujpawar', 'id': 19887541, 'node_id': 'MDQ6VXNlcjE5ODg3NTQx', 'avatar_url': 'https://avatars.githubusercontent.com/u/19887541?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/ambujpawar', 'html_url': 'https://github.com/ambujpawar', 'followers_url': 'https://api.github.com/users/ambujpawar/followers', 'following_url': 'https://api.github.com/users/ambujpawar/following{/other_user}', 'gists_url': 'https://api.github.com/users/ambujpawar/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/ambujpawar/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/ambujpawar/subscriptions', 'organizations_url': 'https://api.github.com/users/ambujpawar/orgs', 'repos_url': 'https://api.github.com/users/ambujpawar/repos', 'events_url': 'https://api.github.com/users/ambujpawar/events{/privacy}', 'received_events_url': 'https://api.github.com/users/ambujpawar/received_events', 'type': 'User', 'site_admin': False}",[],open,False,,[],,0,2022-09-23T13:53:34Z,2022-09-23T13:54:06Z,,CONTRIBUTOR,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48735', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48735', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48735.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48735.patch', 'merged_at': None}","- [x] closes #48395 
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48735/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48735/timeline,,
2,https://api.github.com/repos/pandas-dev/pandas/issues/48734,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48734/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48734/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48734/events,https://github.com/pandas-dev/pandas/pull/48734,1383796561,PR_kwDOAA0YD84_f4Pf,48734,REGR: Raise on invalid colormap for scatter plot,"{'login': 'phofl', 'id': 61934744, 'node_id': 'MDQ6VXNlcjYxOTM0NzQ0', 'avatar_url': 'https://avatars.githubusercontent.com/u/61934744?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/phofl', 'html_url': 'https://github.com/phofl', 'followers_url': 'https://api.github.com/users/phofl/followers', 'following_url': 'https://api.github.com/users/phofl/following{/other_user}', 'gists_url': 'https://api.github.com/users/phofl/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/phofl/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/phofl/subscriptions', 'organizations_url': 'https://api.github.com/users/phofl/orgs', 'repos_url': 'https://api.github.com/users/phofl/repos', 'events_url': 'https://api.github.com/users/phofl/events{/privacy}', 'received_events_url': 'https://api.github.com/users/phofl/received_events', 'type': 'User', 'site_admin': False}","[{'id': 2413328, 'node_id': 'MDU6TGFiZWwyNDEzMzI4', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Visualization', 'name': 'Visualization', 'color': '8AE234', 'default': False, 'description': None}, {'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",0,2022-09-23T13:17:46Z,2022-09-23T14:07:39Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48734', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48734', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48734.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48734.patch', 'merged_at': None}","- [x] closes #48726 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48734/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48734/timeline,,
3,https://api.github.com/repos/pandas-dev/pandas/issues/48733,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48733/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48733/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48733/events,https://github.com/pandas-dev/pandas/issues/48733,1383761105,I_kwDOAA0YD85SeoTR,48733,BUG: AttributeError: 'function' object has no attribute 'currentframe',"{'login': 'huka81', 'id': 114235426, 'node_id': 'U_kgDOBs8YIg', 'avatar_url': 'https://avatars.githubusercontent.com/u/114235426?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/huka81', 'html_url': 'https://github.com/huka81', 'followers_url': 'https://api.github.com/users/huka81/followers', 'following_url': 'https://api.github.com/users/huka81/following{/other_user}', 'gists_url': 'https://api.github.com/users/huka81/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/huka81/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/huka81/subscriptions', 'organizations_url': 'https://api.github.com/users/huka81/orgs', 'repos_url': 'https://api.github.com/users/huka81/repos', 'events_url': 'https://api.github.com/users/huka81/events{/privacy}', 'received_events_url': 'https://api.github.com/users/huka81/received_events', 'type': 'User', 'site_admin': False}","[{'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}, {'id': 47232590, 'node_id': 'MDU6TGFiZWw0NzIzMjU5MA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/IO%20SQL', 'name': 'IO SQL', 'color': '5319e7', 'default': False, 'description': 'to_sql, read_sql, read_sql_query'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",4,2022-09-23T12:51:35Z,2022-09-23T13:48:14Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas.io.sql
import sqlalchemy

dbEngine=sqlalchemy.create_engine(f'sqlite:///dummy.db')

db = pandas.io.sql.SQLDatabase(engine=dbEngine)
db.check_case_sensitive(""TABLE1"", """") # passing
print(""check 1 for existing table passed"")
db.check_case_sensitive(""TABLE2"", """") # failing
print(""check 2 for non-existing table passed"")
```


### Issue Description

Fails on invoking SQLDatabase.check_case_sensitive with table name, which does not exists in database with error:

> Traceback (most recent call last):
>   File ""C:\Users\artur\PycharmProjects\Ex_Files_Python_EssT\Exercise Files\Chap02\pandas_debug.py"", line 15, in <module>
>     db.check_case_sensitive(""TABLE2"", """")
>   File ""C:\Users\artur\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\pandas\io\sql.py"", line 1664, in check_case_sensitive
>     stacklevel=find_stack_level(inspect.currentframe()),
> AttributeError: 'function' object has no attribute 'currentframe'


### Expected Behavior

Same code was passing till version 1.4.4

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.9.13.final.0
python-bits      : 64
OS               : Windows
OS-release       : 10
Version          : 10.0.19044
machine          : AMD64
processor        : Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
byteorder        : little
LC_ALL           : None
LANG             : None
LOCALE           : Polish_Poland.1250

pandas           : 1.5.0
numpy            : 1.21.2
pytz             : 2021.3
dateutil         : 2.8.2
setuptools       : 58.1.0
pip              : 22.0.4
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : None
IPython          : None
pandas_datareader: None
bs4              : None
bottleneck       : None
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : None
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : None
snappy           : None
sqlalchemy       : 1.4.41
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None
tzdata           : None

</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48733/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48733/timeline,,
4,https://api.github.com/repos/pandas-dev/pandas/issues/48729,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48729/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48729/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48729/events,https://github.com/pandas-dev/pandas/issues/48729,1383545853,I_kwDOAA0YD85Sdzv9,48729,BUG: DataFrame.at fails when setting a whole row,"{'login': 'Jeaksen', 'id': 33446723, 'node_id': 'MDQ6VXNlcjMzNDQ2NzIz', 'avatar_url': 'https://avatars.githubusercontent.com/u/33446723?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/Jeaksen', 'html_url': 'https://github.com/Jeaksen', 'followers_url': 'https://api.github.com/users/Jeaksen/followers', 'following_url': 'https://api.github.com/users/Jeaksen/following{/other_user}', 'gists_url': 'https://api.github.com/users/Jeaksen/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/Jeaksen/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/Jeaksen/subscriptions', 'organizations_url': 'https://api.github.com/users/Jeaksen/orgs', 'repos_url': 'https://api.github.com/users/Jeaksen/repos', 'events_url': 'https://api.github.com/users/Jeaksen/events{/privacy}', 'received_events_url': 'https://api.github.com/users/Jeaksen/received_events', 'type': 'User', 'site_admin': False}","[{'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",5,2022-09-23T09:41:57Z,2022-09-23T13:35:08Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas as pd

df = pd.DataFrame(index=['a'], columns=['col1', 'col2'])

df.at['a'] = ['123', '15']
print(df)
```

```python-traceback
Traceback (most recent call last):
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py"", line 3800, in get_loc
    return self._engine.get_loc(casted_key)
  File ""pandas/_libs/index.pyx"", line 138, in pandas._libs.index.IndexEngine.get_loc
  File ""pandas/_libs/index.pyx"", line 144, in pandas._libs.index.IndexEngine.get_loc
TypeError: 'slice(None, None, None)' is an invalid key

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/home/*/pandas_bug.py"", line 6, in <module>
    df.at['a'] = ['123', '15']
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/indexing.py"", line 2438, in __setitem__
    return super().__setitem__(key, value)
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/indexing.py"", line 2393, in __setitem__
    self.obj._set_value(*key, value=value, takeable=self._takeable)
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/frame.py"", line 4208, in _set_value
    icol = self.columns.get_loc(col)
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py"", line 3807, in get_loc
    self._check_indexing_error(key)
  File ""/home/*/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py"", line 5963, in _check_indexing_error
    raise InvalidIndexError(key)
pandas.errors.InvalidIndexError: slice(None, None, None)
```



### Issue Description

Trying to set a row in a DataFrame using DataFrame.at fails on pandas 1.5.0. It worked on pandas 1.4.X.

Setting a single value (i.e. `df.at['a', 'col1'] = '15'`) works fine.
Setting the row with loc works as well (i.e. `df.loc['a'] = ['123', '15']`).

### Expected Behavior

The row values should be set:
```
  col1 col2
a  123   15
```

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.10.4.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.10.16.3-microsoft-standard-WSL2
Version          : #1 SMP Fri Apr 2 22:23:49 UTC 2021
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.5.0
numpy            : 1.23.1
pytz             : 2022.1
dateutil         : 2.8.2
setuptools       : 59.6.0
pip              : 22.0.2
Cython           : 0.29.32
pytest           : 7.1.2
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : 4.8.0
html5lib         : 1.1
pymysql          : None
psycopg2         : 2.9.3
jinja2           : 3.0.3
IPython          : None
pandas_datareader: None
bs4              : 4.10.0
bottleneck       : None
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : None
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : 3.0.10
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.8.1
snappy           : None
sqlalchemy       : 1.4.39
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None
tzdata           : None

</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48729/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48729/timeline,,
5,https://api.github.com/repos/pandas-dev/pandas/issues/48728,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48728/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48728/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48728/events,https://github.com/pandas-dev/pandas/issues/48728,1383349070,I_kwDOAA0YD85SdDtO,48728,QST: handle lines with less separators than main lines in read_csv,"{'login': 'LuisBL', 'id': 694774, 'node_id': 'MDQ6VXNlcjY5NDc3NA==', 'avatar_url': 'https://avatars.githubusercontent.com/u/694774?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/LuisBL', 'html_url': 'https://github.com/LuisBL', 'followers_url': 'https://api.github.com/users/LuisBL/followers', 'following_url': 'https://api.github.com/users/LuisBL/following{/other_user}', 'gists_url': 'https://api.github.com/users/LuisBL/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/LuisBL/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/LuisBL/subscriptions', 'organizations_url': 'https://api.github.com/users/LuisBL/orgs', 'repos_url': 'https://api.github.com/users/LuisBL/repos', 'events_url': 'https://api.github.com/users/LuisBL/events{/privacy}', 'received_events_url': 'https://api.github.com/users/LuisBL/received_events', 'type': 'User', 'site_admin': False}","[{'id': 34444536, 'node_id': 'MDU6TGFiZWwzNDQ0NDUzNg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Usage%20Question', 'name': 'Usage Question', 'color': '0052cc', 'default': False, 'description': None}, {'id': 1954720290, 'node_id': 'MDU6TGFiZWwxOTU0NzIwMjkw', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Needs%20Triage', 'name': 'Needs Triage', 'color': '0052cc', 'default': False, 'description': 'Issue that has not been reviewed by a pandas team member'}]",open,False,,[],,0,2022-09-23T06:26:15Z,2022-09-23T06:26:15Z,,NONE,,,,"### Research

- [X] I have searched the [[pandas] tag](https://stackoverflow.com/questions/tagged/pandas) on StackOverflow for similar questions.

- [X] I have asked my usage related question on [StackOverflow](https://stackoverflow.com).


### Link to question on StackOverflow

https://stackoverflow.com/questions/73820090/make-pandas-read-csv-to-not-add-lines-with-less-columns-delimiters-than-the-main

### Question about pandas

The `on_bad_lines=warn` question me. Pandas team added the functionality to directly handle lines with more separators than the main lines, that's why it don't seems strange to me that some other pandas option could handle in the same way *lines with less separators than main lines*

Using `pandas.read_csv` with `on_bad_lines='warn'` option the case there is too many
columns delimiters work well, bad lines are not loaded and stderr catch the bad lines
numbers:

```python
    import pandas as pd
    from io import StringIO
    data = StringIO(""""""
    nom,f,nb
    bat,F,52
    cat,M,66,
    caw,F,15
    dog,M,66,,
    fly,F,61
    ant,F,21"""""")
    df = pd.read_csv(data, sep=',', on_bad_lines='warn')

    b'Skipping line 4: expected 3 fields, saw 4\nSkipping line 6: expected 3 fields, saw 5\n'

    df.head(10)
    #    nom  f  nb
    # 0  bat  F  52
    # 1  caw  F  15
    # 2  fly  F  61
    # 3  ant  F  21
```

But in case the number of delimiter (here `sep=,`) is less  than the main, the line
is added adding `NaN`.:

```python
    import pandas as pd
    from io import StringIO
    data = StringIO(""""""
    nom,f,nb
    bat,F,52
    catM66,
    caw,F,15
    dog,M66
    fly,F,61
    ant,F,21"""""")
    df = pd.read_csv(data, sep=',', on_bad_lines='warn', dtype=str)
    df.head(10)

    #       nom    f   nb
    # 0     bat    F   52
    # 1  catM66  NaN  NaN            <==
    # 2     caw    F   15
    # 3     dog  M66  NaN            <==
    # 4     fly    F   61
    # 5     ant    F   21
```

Is there a way to make `read_csv` to not add lines with less columns delimiters than
the main lines ?

","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48728/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48728/timeline,,
6,https://api.github.com/repos/pandas-dev/pandas/issues/48726,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48726/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48726/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48726/events,https://github.com/pandas-dev/pandas/issues/48726,1383336725,I_kwDOAA0YD85SdAsV,48726,BUG: DataFrame.plot(kind='scatter') does not raise KeyError when unknown colormap is applied ,"{'login': 'lisphilar', 'id': 7270139, 'node_id': 'MDQ6VXNlcjcyNzAxMzk=', 'avatar_url': 'https://avatars.githubusercontent.com/u/7270139?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/lisphilar', 'html_url': 'https://github.com/lisphilar', 'followers_url': 'https://api.github.com/users/lisphilar/followers', 'following_url': 'https://api.github.com/users/lisphilar/following{/other_user}', 'gists_url': 'https://api.github.com/users/lisphilar/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/lisphilar/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/lisphilar/subscriptions', 'organizations_url': 'https://api.github.com/users/lisphilar/orgs', 'repos_url': 'https://api.github.com/users/lisphilar/repos', 'events_url': 'https://api.github.com/users/lisphilar/events{/privacy}', 'received_events_url': 'https://api.github.com/users/lisphilar/received_events', 'type': 'User', 'site_admin': False}","[{'id': 2413328, 'node_id': 'MDU6TGFiZWwyNDEzMzI4', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Visualization', 'name': 'Visualization', 'color': '8AE234', 'default': False, 'description': None}, {'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",1,2022-09-23T06:07:46Z,2022-09-23T13:19:48Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [ ] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas as pd
iris = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv')
iris.plot(x='sepal_length', y='sepal_width', colormap='unknown')
# -> TypeError raised as expected
iris.plot(x='sepal_length', y='sepal_width', colormap='unknown', kind='scatter')
# -> TypeError is expected but not raised at version 1.5.0
```


### Issue Description

`.plot(..., colormap='unknown')` raises `TypeError` because the colormap is unknown, but `.plot(..., colormap='unknown', kind='scatter')` does raise `TypeError`.

### Expected Behavior

`.plot(..., cmap='unknown', kind='scatter')` will raise `TypeError`.

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.10.7.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.10.16.3-microsoft-standard-WSL2
Version          : #1 SMP Fri Apr 2 22:23:49 UTC 2021
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.5.0
numpy            : 1.23.3
pytz             : 2022.2.1
dateutil         : 2.8.2
setuptools       : 65.3.0
pip              : 22.2.2
Cython           : None
pytest           : 7.1.3
hypothesis       : None
sphinx           : 5.1.1
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : 4.9.1
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 3.1.2
IPython          : 8.5.0
pandas_datareader: None
bs4              : 4.11.1
bottleneck       : None
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : 3.6.0
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : 9.0.0
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.8.1
snappy           : None
sqlalchemy       : 1.4.41
tables           : None
tabulate         : 0.8.10
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None
tzdata           : None

</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48726/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48726/timeline,,
7,https://api.github.com/repos/pandas-dev/pandas/issues/48723,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48723/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48723/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48723/events,https://github.com/pandas-dev/pandas/pull/48723,1383139438,PR_kwDOAA0YD84_dvLO,48723,PERF: MultiIndex.size,"{'login': 'lukemanley', 'id': 8519523, 'node_id': 'MDQ6VXNlcjg1MTk1MjM=', 'avatar_url': 'https://avatars.githubusercontent.com/u/8519523?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/lukemanley', 'html_url': 'https://github.com/lukemanley', 'followers_url': 'https://api.github.com/users/lukemanley/followers', 'following_url': 'https://api.github.com/users/lukemanley/following{/other_user}', 'gists_url': 'https://api.github.com/users/lukemanley/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/lukemanley/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/lukemanley/subscriptions', 'organizations_url': 'https://api.github.com/users/lukemanley/orgs', 'repos_url': 'https://api.github.com/users/lukemanley/repos', 'events_url': 'https://api.github.com/users/lukemanley/events{/privacy}', 'received_events_url': 'https://api.github.com/users/lukemanley/received_events', 'type': 'User', 'site_admin': False}","[{'id': 8935311, 'node_id': 'MDU6TGFiZWw4OTM1MzEx', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Performance', 'name': 'Performance', 'color': 'a10c02', 'default': False, 'description': 'Memory or execution speed performance'}, {'id': 71268330, 'node_id': 'MDU6TGFiZWw3MTI2ODMzMA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/MultiIndex', 'name': 'MultiIndex', 'color': '207de5', 'default': False, 'description': None}]",open,False,,[],,0,2022-09-23T00:04:06Z,2022-09-23T00:05:27Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48723', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48723', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48723.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48723.patch', 'merged_at': None}","- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/v1.6.0.rst` file if fixing a bug or adding a new feature.


Improvement comes from avoiding MultiIndex._values

```
import pandas as pd
import numpy as np

mi = pd.MultiIndex.from_product(
    [
        pd.date_range('1970-01-01', periods=1000),
        np.arange(1000),
    ]
)

%timeit mi.copy().size
```

```
111 ms ± 9.11 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)       <- main
17.7 µs ± 1.1 µs per loop (mean ± std. dev. of 7 runs, 100,000 loops each)  <- PR
```

At least one existing asv shows improvement (there may be others):

```
 before           after         ratio
     [cda0f6be]       [8395c77a]
     <main>           <multiindex-size>
-      25.1±0.4ms         18.3±1ms     0.73  multiindex_object.SetOperations.time_operation('non_monotonic', 'string', 'symmetric_difference')
-      24.8±0.4ms         16.7±2ms     0.68  multiindex_object.SetOperations.time_operation('monotonic', 'ea_int', 'symmetric_difference')
-      24.4±0.8ms       15.8±0.3ms     0.65  multiindex_object.SetOperations.time_operation('monotonic', 'string', 'symmetric_difference')
-      25.3±0.4ms         16.4±1ms     0.65  multiindex_object.SetOperations.time_operation('non_monotonic', 'ea_int', 'symmetric_difference')
-      23.9±0.4ms       13.5±0.2ms     0.57  multiindex_object.SetOperations.time_operation('non_monotonic', 'datetime', 'symmetric_difference')
-      23.0±0.6ms         12.1±2ms     0.53  multiindex_object.SetOperations.time_operation('non_monotonic', 'int', 'symmetric_difference')
-        25.1±1ms       13.0±0.6ms     0.52  multiindex_object.SetOperations.time_operation('monotonic', 'datetime', 'symmetric_difference')
-      23.0±0.4ms       11.2±0.6ms     0.49  multiindex_object.SetOperations.time_operation('monotonic', 'int', 'symmetric_difference')
```
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48723/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48723/timeline,,
8,https://api.github.com/repos/pandas-dev/pandas/issues/48721,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48721/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48721/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48721/events,https://github.com/pandas-dev/pandas/issues/48721,1383102326,I_kwDOAA0YD85ScHd2,48721,BUG/TST: clipboard tests xpassing on mac,"{'login': 'jbrockmendel', 'id': 8078968, 'node_id': 'MDQ6VXNlcjgwNzg5Njg=', 'avatar_url': 'https://avatars.githubusercontent.com/u/8078968?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jbrockmendel', 'html_url': 'https://github.com/jbrockmendel', 'followers_url': 'https://api.github.com/users/jbrockmendel/followers', 'following_url': 'https://api.github.com/users/jbrockmendel/following{/other_user}', 'gists_url': 'https://api.github.com/users/jbrockmendel/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jbrockmendel/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jbrockmendel/subscriptions', 'organizations_url': 'https://api.github.com/users/jbrockmendel/orgs', 'repos_url': 'https://api.github.com/users/jbrockmendel/repos', 'events_url': 'https://api.github.com/users/jbrockmendel/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jbrockmendel/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76811, 'node_id': 'MDU6TGFiZWw3NjgxMQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Bug', 'name': 'Bug', 'color': 'e10c02', 'default': False, 'description': None}, {'id': 1954720290, 'node_id': 'MDU6TGFiZWwxOTU0NzIwMjkw', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Needs%20Triage', 'name': 'Needs Triage', 'color': '0052cc', 'default': False, 'description': 'Issue that has not been reviewed by a pandas team member'}]",open,False,,[],,0,2022-09-22T22:52:54Z,2022-09-22T22:53:08Z,,MEMBER,,,,"test_raw_roundtrip xpassing on mac, py39, main
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48721/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48721/timeline,,
9,https://api.github.com/repos/pandas-dev/pandas/issues/48720,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48720/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48720/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48720/events,https://github.com/pandas-dev/pandas/pull/48720,1383043105,PR_kwDOAA0YD84_dbKr,48720,DOC: re-phrase web README.md ENG,"{'login': 'anonmouse1', 'id': 90865869, 'node_id': 'MDQ6VXNlcjkwODY1ODY5', 'avatar_url': 'https://avatars.githubusercontent.com/u/90865869?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/anonmouse1', 'html_url': 'https://github.com/anonmouse1', 'followers_url': 'https://api.github.com/users/anonmouse1/followers', 'following_url': 'https://api.github.com/users/anonmouse1/following{/other_user}', 'gists_url': 'https://api.github.com/users/anonmouse1/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/anonmouse1/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/anonmouse1/subscriptions', 'organizations_url': 'https://api.github.com/users/anonmouse1/orgs', 'repos_url': 'https://api.github.com/users/anonmouse1/repos', 'events_url': 'https://api.github.com/users/anonmouse1/events{/privacy}', 'received_events_url': 'https://api.github.com/users/anonmouse1/received_events', 'type': 'User', 'site_admin': False}",[],open,False,,[],,0,2022-09-22T21:35:21Z,2022-09-22T21:35:21Z,,NONE,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48720', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48720', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48720.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48720.patch', 'merged_at': None}","Basic rephrasing of the web README.md for improved legibility 
- ""to navigate it "" -> ""to allow navigation of the website "" (""it"" is ambiguous here, ""website"" leaves no question as to which entity the instructions refer to.
- ""it is needed"" -> ""must""
- ""a not open"" -> ""it is not possible to open "" (perhaps a typo here initially, sentence lengthened for improved legibility)



","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48720/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48720/timeline,,
10,https://api.github.com/repos/pandas-dev/pandas/issues/48719,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48719/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48719/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48719/events,https://github.com/pandas-dev/pandas/pull/48719,1382991084,PR_kwDOAA0YD84_dQZr,48719,CI: include tests in coverage,"{'login': 'MarcoGorelli', 'id': 33491632, 'node_id': 'MDQ6VXNlcjMzNDkxNjMy', 'avatar_url': 'https://avatars.githubusercontent.com/u/33491632?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/MarcoGorelli', 'html_url': 'https://github.com/MarcoGorelli', 'followers_url': 'https://api.github.com/users/MarcoGorelli/followers', 'following_url': 'https://api.github.com/users/MarcoGorelli/following{/other_user}', 'gists_url': 'https://api.github.com/users/MarcoGorelli/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/MarcoGorelli/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/MarcoGorelli/subscriptions', 'organizations_url': 'https://api.github.com/users/MarcoGorelli/orgs', 'repos_url': 'https://api.github.com/users/MarcoGorelli/repos', 'events_url': 'https://api.github.com/users/MarcoGorelli/events{/privacy}', 'received_events_url': 'https://api.github.com/users/MarcoGorelli/received_events', 'type': 'User', 'site_admin': False}",[],open,False,,[],,0,2022-09-22T20:41:15Z,2022-09-22T20:41:15Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48719', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48719', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48719.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48719.patch', 'merged_at': None}","It's hard to tell whether all tests are being executed in CI - can we add not omit them from the coverage report, so this'll be easier?","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48719/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48719/timeline,,
11,https://api.github.com/repos/pandas-dev/pandas/issues/48715,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48715/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48715/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48715/events,https://github.com/pandas-dev/pandas/issues/48715,1382647325,I_kwDOAA0YD85SaYYd,48715,"DOC: ""Show Source"" button on API pages does not link to expected source code","{'login': 'tehunter', 'id': 7980666, 'node_id': 'MDQ6VXNlcjc5ODA2NjY=', 'avatar_url': 'https://avatars.githubusercontent.com/u/7980666?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/tehunter', 'html_url': 'https://github.com/tehunter', 'followers_url': 'https://api.github.com/users/tehunter/followers', 'following_url': 'https://api.github.com/users/tehunter/following{/other_user}', 'gists_url': 'https://api.github.com/users/tehunter/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/tehunter/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/tehunter/subscriptions', 'organizations_url': 'https://api.github.com/users/tehunter/orgs', 'repos_url': 'https://api.github.com/users/tehunter/repos', 'events_url': 'https://api.github.com/users/tehunter/events{/privacy}', 'received_events_url': 'https://api.github.com/users/tehunter/received_events', 'type': 'User', 'site_admin': False}","[{'id': 134699, 'node_id': 'MDU6TGFiZWwxMzQ2OTk=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Docs', 'name': 'Docs', 'color': '3465A4', 'default': False, 'description': None}, {'id': 1954720290, 'node_id': 'MDU6TGFiZWwxOTU0NzIwMjkw', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Needs%20Triage', 'name': 'Needs Triage', 'color': '0052cc', 'default': False, 'description': 'Issue that has not been reviewed by a pandas team member'}]",open,False,,[],,0,2022-09-22T15:33:53Z,2022-09-22T15:33:53Z,,CONTRIBUTOR,,,,"### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)


### Location of the documentation

https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.html

### Documentation problem

The ""Show Source"" button on the new version of the documentation pages is not useful to most users and is likely to cause confusion. In addition, it conflicts with the ""[source]"" link which also exists on the page (and on previous versions of the documentation).

![image](https://user-images.githubusercontent.com/7980666/191789465-ad1cd25c-2912-405e-9f54-07a7728266f7.png)

Rather than showing the relevant API source code, the ""Show Source"" button shows the source code for the current documentation page (.e.g. https://pandas.pydata.org/docs/_sources/reference/api/pandas.DataFrame.rst.txt).

### Suggested fix for documentation

I suggest the ""Show Source"" button be removed, or it should be remapped to the same link as the current ""[source]"" link (which should then be removed).

I could see some value for developers wanted to access the documentation source. If that is important, then I would propose (1) move to the bottom or out of a high-visibility area, and (2) rename it to ""Show Documentation Source""","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48715/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48715/timeline,,
12,https://api.github.com/repos/pandas-dev/pandas/issues/48713,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48713/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48713/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48713/events,https://github.com/pandas-dev/pandas/pull/48713,1382480814,PR_kwDOAA0YD84_blvE,48713,BUG: pivot_table raising Future Warning with datetime column as index,"{'login': 'phofl', 'id': 61934744, 'node_id': 'MDQ6VXNlcjYxOTM0NzQ0', 'avatar_url': 'https://avatars.githubusercontent.com/u/61934744?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/phofl', 'html_url': 'https://github.com/phofl', 'followers_url': 'https://api.github.com/users/phofl/followers', 'following_url': 'https://api.github.com/users/phofl/following{/other_user}', 'gists_url': 'https://api.github.com/users/phofl/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/phofl/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/phofl/subscriptions', 'organizations_url': 'https://api.github.com/users/phofl/orgs', 'repos_url': 'https://api.github.com/users/phofl/repos', 'events_url': 'https://api.github.com/users/phofl/events{/privacy}', 'received_events_url': 'https://api.github.com/users/phofl/received_events', 'type': 'User', 'site_admin': False}","[{'id': 13098779, 'node_id': 'MDU6TGFiZWwxMzA5ODc3OQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Reshaping', 'name': 'Reshaping', 'color': '02d7e1', 'default': False, 'description': 'Concat, Merge/Join, Stack/Unstack, Explode'}, {'id': 1628184320, 'node_id': 'MDU6TGFiZWwxNjI4MTg0MzIw', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Warnings', 'name': 'Warnings', 'color': 'f2f074', 'default': False, 'description': 'Warnings that appear or should be added to pandas'}]",open,False,,[],,0,2022-09-22T13:44:43Z,2022-09-22T20:37:33Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48713', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48713', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48713.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48713.patch', 'merged_at': None}","- [x] closes #48683 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

I think I'd prefer back porting, but no strong opinion","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48713/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48713/timeline,,
13,https://api.github.com/repos/pandas-dev/pandas/issues/48712,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48712/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48712/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48712/events,https://github.com/pandas-dev/pandas/issues/48712,1382462765,I_kwDOAA0YD85SZrUt,48712,BUG: read_csv function bug - null,"{'login': 'Rand0x', 'id': 81554629, 'node_id': 'MDQ6VXNlcjgxNTU0NjI5', 'avatar_url': 'https://avatars.githubusercontent.com/u/81554629?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/Rand0x', 'html_url': 'https://github.com/Rand0x', 'followers_url': 'https://api.github.com/users/Rand0x/followers', 'following_url': 'https://api.github.com/users/Rand0x/following{/other_user}', 'gists_url': 'https://api.github.com/users/Rand0x/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/Rand0x/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/Rand0x/subscriptions', 'organizations_url': 'https://api.github.com/users/Rand0x/orgs', 'repos_url': 'https://api.github.com/users/Rand0x/repos', 'events_url': 'https://api.github.com/users/Rand0x/events{/privacy}', 'received_events_url': 'https://api.github.com/users/Rand0x/received_events', 'type': 'User', 'site_admin': False}","[{'id': 47229171, 'node_id': 'MDU6TGFiZWw0NzIyOTE3MQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/IO%20CSV', 'name': 'IO CSV', 'color': '5319e7', 'default': False, 'description': 'read_csv, to_csv'}, {'id': 307649777, 'node_id': 'MDU6TGFiZWwzMDc2NDk3Nzc=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Needs%20Info', 'name': 'Needs Info', 'color': '207de5', 'default': False, 'description': 'Clarification about behavior needed to assess issue'}]",open,False,,[],,1,2022-09-22T13:32:31Z,2022-09-22T13:46:31Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [ ] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas as pd
import numpy as np
np.set_printoptions(precision=3, suppress=True)
import os
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
import tensorflow
import tensorflow.keras import layers #tensorflowversion = 2.4.0

df = pd.read.csv('data/train.csv')
pd.set_option('display.max_rows', df.shape[0]+1)
print(df)
#null = NaN
```


### Issue Description

In my dataset are just some values and a colum with label as name.
One label has the name ""null""
This converts the null in NaN which destroys my dataset completly.

### Expected Behavior

Expected Behavior is that the string null is still ""null"" and not ""NaN""

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.8.10.final.0
python-bits      : 64
OS               : Linux
OS-release       : 3.10.0-1160.25.1.el7.x86_64
Version          : #1 SMP Tue Apr 13 18:55:45 EDT 2021
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.5.0
numpy            : 1.23.3
pytz             : 2022.2.1
dateutil         : 2.8.2
setuptools       : 62.3.2
pip              : 20.2.4
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : None
IPython          : None
pandas_datareader: None
bs4              : None
bottleneck       : None
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : None
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.9.1
snappy           : None
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None
tzdata           : None
None


</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48712/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48712/timeline,,
14,https://api.github.com/repos/pandas-dev/pandas/issues/48706,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48706/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48706/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48706/events,https://github.com/pandas-dev/pandas/issues/48706,1382018640,I_kwDOAA0YD85SX-5Q,48706,BUG: Excel import crashes on simple file,"{'login': 'apacha', 'id': 594497, 'node_id': 'MDQ6VXNlcjU5NDQ5Nw==', 'avatar_url': 'https://avatars.githubusercontent.com/u/594497?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/apacha', 'html_url': 'https://github.com/apacha', 'followers_url': 'https://api.github.com/users/apacha/followers', 'following_url': 'https://api.github.com/users/apacha/following{/other_user}', 'gists_url': 'https://api.github.com/users/apacha/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/apacha/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/apacha/subscriptions', 'organizations_url': 'https://api.github.com/users/apacha/orgs', 'repos_url': 'https://api.github.com/users/apacha/repos', 'events_url': 'https://api.github.com/users/apacha/events{/privacy}', 'received_events_url': 'https://api.github.com/users/apacha/received_events', 'type': 'User', 'site_admin': False}","[{'id': 49254273, 'node_id': 'MDU6TGFiZWw0OTI1NDI3Mw==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/IO%20Excel', 'name': 'IO Excel', 'color': 'bfe5bf', 'default': False, 'description': 'read_excel, to_excel'}, {'id': 2365504893, 'node_id': 'MDU6TGFiZWwyMzY1NTA0ODkz', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Closing%20Candidate', 'name': 'Closing Candidate', 'color': 'be21f2', 'default': False, 'description': 'May be closeable, needs more eyeballs'}]",open,False,,[],,5,2022-09-22T08:09:26Z,2022-09-22T14:10:42Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [x] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas as pd
file = pd.read_excel(""/Users/alex/Desktop/BrokenExcel.xlsx"", sheet_name=None, header=1)
```


### Issue Description

Loading the attached file produces an exception and crashes in 1.5.0 but worked in 1.4.3.

```bash
venv/bin/python /Users/alex/Library/Application Support/JetBrains/PyCharm2022.2/scratches/scratch_5.py 
Traceback (most recent call last):
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/python_parser.py"", line 701, in _next_line
    line = self._check_comments([self.data[self.pos]])[0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/python_parser.py"", line 386, in _infer_columns
    line = self._next_line()
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/python_parser.py"", line 714, in _next_line
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File ""/Users/alex/Library/Application Support/JetBrains/PyCharm2022.2/scratches/scratch_5.py"", line 2, in <module>
    file = pd.read_excel(""/Users/alex/Desktop/BrokenExcel.xlsx"", sheet_name=None, header=1)
  File ""venv/lib/python3.9/site-packages/pandas/util/_decorators.py"", line 211, in wrapper
    return func(*args, **kwargs)
  File ""venv/lib/python3.9/site-packages/pandas/util/_decorators.py"", line 317, in wrapper
    return func(*args, **kwargs)
  File ""venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 491, in read_excel
    data = io.parse(
  File ""venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 1711, in parse
    return self._reader.parse(
  File ""venv/lib/python3.9/site-packages/pandas/io/excel/_base.py"", line 867, in parse
    parser = TextParser(
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py"", line 1870, in TextParser
    return TextFileReader(*args, **kwds)
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py"", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py"", line 1747, in _make_engine
    return mapping[engine](f, **self.options)
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/python_parser.py"", line 123, in __init__
    ) = self._infer_columns()
  File ""venv/lib/python3.9/site-packages/pandas/io/parsers/python_parser.py"", line 396, in _infer_columns
    raise ValueError(
ValueError: Passed header=[1], len of 1, but only 1 lines in file

Process finished with exit code 1

```

### Expected Behavior

Loading of the Excel file (as in 1.4.3)

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.9.13.final.0
python-bits      : 64
OS               : Darwin
OS-release       : 21.6.0
Version          : Darwin Kernel Version 21.6.0: Mon Aug 22 20:17:10 PDT 2022; root:xnu-8020.140.49~2/RELEASE_X86_64
machine          : x86_64
processor        : i386
byteorder        : little
LC_ALL           : None
LANG             : None
LOCALE           : en_US.UTF-8
pandas           : 1.5.0
numpy            : 1.23.3
pytz             : 2022.1
dateutil         : 2.8.2
setuptools       : 62.6.0
pip              : 22.2.2
Cython           : None
pytest           : 7.1.3
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : 4.9.1
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 3.1.2
IPython          : None
pandas_datareader: None
bs4              : None
bottleneck       : None
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : None
numba            : None
numexpr          : None
odfpy            : None
openpyxl         : 3.0.10
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.9.0
snappy           : None
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
zstandard        : None
tzdata           : None

</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48706/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48706/timeline,,reopened
15,https://api.github.com/repos/pandas-dev/pandas/issues/48705,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48705/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48705/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48705/events,https://github.com/pandas-dev/pandas/pull/48705,1382009681,PR_kwDOAA0YD84_aDNT,48705,Handle NaN in array_with_unit_to_datetime,"{'login': 'andreas-schwab', 'id': 2175493, 'node_id': 'MDQ6VXNlcjIxNzU0OTM=', 'avatar_url': 'https://avatars.githubusercontent.com/u/2175493?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/andreas-schwab', 'html_url': 'https://github.com/andreas-schwab', 'followers_url': 'https://api.github.com/users/andreas-schwab/followers', 'following_url': 'https://api.github.com/users/andreas-schwab/following{/other_user}', 'gists_url': 'https://api.github.com/users/andreas-schwab/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/andreas-schwab/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/andreas-schwab/subscriptions', 'organizations_url': 'https://api.github.com/users/andreas-schwab/orgs', 'repos_url': 'https://api.github.com/users/andreas-schwab/repos', 'events_url': 'https://api.github.com/users/andreas-schwab/events{/privacy}', 'received_events_url': 'https://api.github.com/users/andreas-schwab/received_events', 'type': 'User', 'site_admin': False}","[{'id': 211840, 'node_id': 'MDU6TGFiZWwyMTE4NDA=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timeseries', 'name': 'Timeseries', 'color': 'AFEEEE', 'default': False, 'description': None}, {'id': 2822342, 'node_id': 'MDU6TGFiZWwyODIyMzQy', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Missing-data', 'name': 'Missing-data', 'color': 'd7e102', 'default': False, 'description': 'np.nan, pd.NaT, pd.NA, dropna, isnull, interpolate'}]",open,False,,[],,6,2022-09-22T08:02:39Z,2022-09-22T17:26:26Z,,NONE,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48705', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48705', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48705.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48705.patch', 'merged_at': None}","Missing values in arrays can either be NaT (for integer types) or NaN (for
floating types).  Make sure to also handle the latter.  This fixes all
failing tests.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48705/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48705/timeline,,
16,https://api.github.com/repos/pandas-dev/pandas/issues/48703,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48703/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48703/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48703/events,https://github.com/pandas-dev/pandas/pull/48703,1381836929,PR_kwDOAA0YD84_ZfkG,48703,Added theme_switcher ,"{'login': 'deepaksirohiwal', 'id': 38135521, 'node_id': 'MDQ6VXNlcjM4MTM1NTIx', 'avatar_url': 'https://avatars.githubusercontent.com/u/38135521?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/deepaksirohiwal', 'html_url': 'https://github.com/deepaksirohiwal', 'followers_url': 'https://api.github.com/users/deepaksirohiwal/followers', 'following_url': 'https://api.github.com/users/deepaksirohiwal/following{/other_user}', 'gists_url': 'https://api.github.com/users/deepaksirohiwal/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/deepaksirohiwal/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/deepaksirohiwal/subscriptions', 'organizations_url': 'https://api.github.com/users/deepaksirohiwal/orgs', 'repos_url': 'https://api.github.com/users/deepaksirohiwal/repos', 'events_url': 'https://api.github.com/users/deepaksirohiwal/events{/privacy}', 'received_events_url': 'https://api.github.com/users/deepaksirohiwal/received_events', 'type': 'User', 'site_admin': False}","[{'id': 1508144531, 'node_id': 'MDU6TGFiZWwxNTA4MTQ0NTMx', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Web', 'name': 'Web', 'color': '23ef97', 'default': False, 'description': 'pandas website'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",3,2022-09-22T05:02:44Z,2022-09-23T10:30:15Z,,NONE,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48703', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48703', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48703.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48703.patch', 'merged_at': None}","Added theme_switcher in  navbar_end to display the Dark/Light theme switcher

- [x] closes #48598 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48703/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48703/timeline,,
17,https://api.github.com/repos/pandas-dev/pandas/issues/48702,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48702/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48702/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48702/events,https://github.com/pandas-dev/pandas/pull/48702,1381665639,PR_kwDOAA0YD84_Y8D5,48702,REGR: dropna affects observed in groupby,"{'login': 'rhshadrach', 'id': 45562402, 'node_id': 'MDQ6VXNlcjQ1NTYyNDAy', 'avatar_url': 'https://avatars.githubusercontent.com/u/45562402?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/rhshadrach', 'html_url': 'https://github.com/rhshadrach', 'followers_url': 'https://api.github.com/users/rhshadrach/followers', 'following_url': 'https://api.github.com/users/rhshadrach/following{/other_user}', 'gists_url': 'https://api.github.com/users/rhshadrach/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/rhshadrach/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/rhshadrach/subscriptions', 'organizations_url': 'https://api.github.com/users/rhshadrach/orgs', 'repos_url': 'https://api.github.com/users/rhshadrach/repos', 'events_url': 'https://api.github.com/users/rhshadrach/events{/privacy}', 'received_events_url': 'https://api.github.com/users/rhshadrach/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76811, 'node_id': 'MDU6TGFiZWw3NjgxMQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Bug', 'name': 'Bug', 'color': 'e10c02', 'default': False, 'description': None}, {'id': 233160, 'node_id': 'MDU6TGFiZWwyMzMxNjA=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Groupby', 'name': 'Groupby', 'color': '729FCF', 'default': False, 'description': None}, {'id': 2822342, 'node_id': 'MDU6TGFiZWwyODIyMzQy', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Missing-data', 'name': 'Missing-data', 'color': 'd7e102', 'default': False, 'description': 'np.nan, pd.NaT, pd.NA, dropna, isnull, interpolate'}, {'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}, {'id': 78527356, 'node_id': 'MDU6TGFiZWw3ODUyNzM1Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Categorical', 'name': 'Categorical', 'color': 'e11d21', 'default': False, 'description': 'Categorical Data Type'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",0,2022-09-22T00:43:53Z,2022-09-23T00:49:59Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48702', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48702', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48702.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48702.patch', 'merged_at': None}","- [x] closes #48645 (Replace xxxx with the Github issue number)
- [x] closes #46909 (the cleanup is no longer applicable) 
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

Since this reintroduces a bug, a longer whatsnew note seemed appropriate.","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48702/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48702/timeline,,
18,https://api.github.com/repos/pandas-dev/pandas/issues/48700,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48700/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48700/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48700/events,https://github.com/pandas-dev/pandas/issues/48700,1381626146,I_kwDOAA0YD85SWfEi,48700,StataReader processes whole file before reading in chunks,"{'login': 'sterlinm', 'id': 1461273, 'node_id': 'MDQ6VXNlcjE0NjEyNzM=', 'avatar_url': 'https://avatars.githubusercontent.com/u/1461273?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/sterlinm', 'html_url': 'https://github.com/sterlinm', 'followers_url': 'https://api.github.com/users/sterlinm/followers', 'following_url': 'https://api.github.com/users/sterlinm/following{/other_user}', 'gists_url': 'https://api.github.com/users/sterlinm/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/sterlinm/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/sterlinm/subscriptions', 'organizations_url': 'https://api.github.com/users/sterlinm/orgs', 'repos_url': 'https://api.github.com/users/sterlinm/repos', 'events_url': 'https://api.github.com/users/sterlinm/events{/privacy}', 'received_events_url': 'https://api.github.com/users/sterlinm/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76812, 'node_id': 'MDU6TGFiZWw3NjgxMg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Enhancement', 'name': 'Enhancement', 'color': '4E9A06', 'default': False, 'description': None}, {'id': 104865385, 'node_id': 'MDU6TGFiZWwxMDQ4NjUzODU=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/IO%20Stata', 'name': 'IO Stata', 'color': '5319e7', 'default': False, 'description': 'read_stata, to_stata'}]",open,False,,[],,4,2022-09-21T23:26:44Z,2022-09-23T02:41:55Z,,NONE,,,,"I've noticed that when reading large Stata files using the chunksize parameter the time it takes to create the StataReader object is affected by the size of the file. This is a bit surprising since all of the metadata it needs is contained in the file header so it seems like it should take the same time regardless of the total file size.

I took a look at the code and it seems like the culprit is this line that reads the entire file into a BytesIO object before parsing the header. I'm not entirely sure what this accomplishes. Ideally it would be nice to be able to create the StataReader object after processing just the header portion of the file.

https://github.com/pandas-dev/pandas/blob/71fc89cd515c3c19230fbab64e979118858b808a/pandas/io/stata.py#L1167-L1175","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48700/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48700/timeline,,
19,https://api.github.com/repos/pandas-dev/pandas/issues/48698,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48698/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48698/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48698/events,https://github.com/pandas-dev/pandas/pull/48698,1381550916,PR_kwDOAA0YD84_Yjdv,48698,CLN: Clean groupby ops from unreached code paths,"{'login': 'phofl', 'id': 61934744, 'node_id': 'MDQ6VXNlcjYxOTM0NzQ0', 'avatar_url': 'https://avatars.githubusercontent.com/u/61934744?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/phofl', 'html_url': 'https://github.com/phofl', 'followers_url': 'https://api.github.com/users/phofl/followers', 'following_url': 'https://api.github.com/users/phofl/following{/other_user}', 'gists_url': 'https://api.github.com/users/phofl/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/phofl/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/phofl/subscriptions', 'organizations_url': 'https://api.github.com/users/phofl/orgs', 'repos_url': 'https://api.github.com/users/phofl/repos', 'events_url': 'https://api.github.com/users/phofl/events{/privacy}', 'received_events_url': 'https://api.github.com/users/phofl/received_events', 'type': 'User', 'site_admin': False}","[{'id': 233160, 'node_id': 'MDU6TGFiZWwyMzMxNjA=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Groupby', 'name': 'Groupby', 'color': '729FCF', 'default': False, 'description': None}, {'id': 211029535, 'node_id': 'MDU6TGFiZWwyMTEwMjk1MzU=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Clean', 'name': 'Clean', 'color': '207de5', 'default': False, 'description': None}]",open,False,,[],,1,2022-09-21T21:55:49Z,2022-09-22T20:38:39Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48698', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48698', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48698.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48698.patch', 'merged_at': None}","- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

We can't hit these code paths anymore","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48698/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48698/timeline,,
20,https://api.github.com/repos/pandas-dev/pandas/issues/48697,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48697/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48697/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48697/events,https://github.com/pandas-dev/pandas/pull/48697,1381550181,PR_kwDOAA0YD84_YjTP,48697,REGR: None converted to NaN when enlarging Series,"{'login': 'phofl', 'id': 61934744, 'node_id': 'MDQ6VXNlcjYxOTM0NzQ0', 'avatar_url': 'https://avatars.githubusercontent.com/u/61934744?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/phofl', 'html_url': 'https://github.com/phofl', 'followers_url': 'https://api.github.com/users/phofl/followers', 'following_url': 'https://api.github.com/users/phofl/following{/other_user}', 'gists_url': 'https://api.github.com/users/phofl/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/phofl/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/phofl/subscriptions', 'organizations_url': 'https://api.github.com/users/phofl/orgs', 'repos_url': 'https://api.github.com/users/phofl/repos', 'events_url': 'https://api.github.com/users/phofl/events{/privacy}', 'received_events_url': 'https://api.github.com/users/phofl/received_events', 'type': 'User', 'site_admin': False}","[{'id': 2822098, 'node_id': 'MDU6TGFiZWwyODIyMDk4', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Indexing', 'name': 'Indexing', 'color': '0b02e1', 'default': False, 'description': 'Related to indexing on series/frames, not to indexes themselves'}, {'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",0,2022-09-21T21:55:12Z,2022-09-22T20:40:02Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48697', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48697', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48697.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48697.patch', 'merged_at': None}","- [x] closes #48665 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48697/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48697/timeline,,
21,https://api.github.com/repos/pandas-dev/pandas/issues/48694,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48694/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48694/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48694/events,https://github.com/pandas-dev/pandas/issues/48694,1381498406,I_kwDOAA0YD85SV_4m,48694,BUG: UndefinedVariableError in DataFrame.query if name is type,"{'login': 'pm13', 'id': 6249977, 'node_id': 'MDQ6VXNlcjYyNDk5Nzc=', 'avatar_url': 'https://avatars.githubusercontent.com/u/6249977?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/pm13', 'html_url': 'https://github.com/pm13', 'followers_url': 'https://api.github.com/users/pm13/followers', 'following_url': 'https://api.github.com/users/pm13/following{/other_user}', 'gists_url': 'https://api.github.com/users/pm13/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/pm13/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/pm13/subscriptions', 'organizations_url': 'https://api.github.com/users/pm13/orgs', 'repos_url': 'https://api.github.com/users/pm13/repos', 'events_url': 'https://api.github.com/users/pm13/events{/privacy}', 'received_events_url': 'https://api.github.com/users/pm13/received_events', 'type': 'User', 'site_admin': False}","[{'id': 32815646, 'node_id': 'MDU6TGFiZWwzMjgxNTY0Ng==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Regression', 'name': 'Regression', 'color': 'e11d21', 'default': False, 'description': 'Functionality that used to work in a prior pandas version'}, {'id': 1631985267, 'node_id': 'MDU6TGFiZWwxNjMxOTg1MjY3', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/expressions', 'name': 'expressions', 'color': 'f3afff', 'default': False, 'description': 'pd.eval, query'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",2,2022-09-21T21:13:26Z,2022-09-21T23:16:33Z,,NONE,,,,"### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the main branch of pandas.


### Reproducible Example

```python
import pandas as pd

df = pd.DataFrame({'x': [1, 2, 3], 'y': [4, 5, 6]})

class A:
    a = 1

df.query('x == @A.a')  # <-- this produces the error
```


### Issue Description

In Pandas 1.5.0 it is not possible to use class as name in DataFrame.query:

<details>

```python
>>> import pandas as pd
>>> 
>>> df = pd.DataFrame({'x': [1, 2, 3], 'y': [4, 5, 6]})
>>> 
>>> class A:
...     a = 1
... 
>>> df.query('x == @A.a')  # <-- this produces the error
Traceback (most recent call last):
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/scope.py"", line 198, in resolve
    return self.resolvers[key]
  File ""/usr/lib64/python3.10/collections/__init__.py"", line 986, in __getitem__
    return self.__missing__(key)            # support subclasses that define __missing__
  File ""/usr/lib64/python3.10/collections/__init__.py"", line 978, in __missing__
    raise KeyError(key)
KeyError: 'A'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/scope.py"", line 209, in resolve
    return self.temps[key]
KeyError: 'A'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File ""<stdin>"", line 1, in <module>
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/util/_decorators.py"", line 317, in wrapper
    return func(*args, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/frame.py"", line 4464, in query
    res = self.eval(expr, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/util/_decorators.py"", line 317, in wrapper
    return func(*args, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/frame.py"", line 4602, in eval
    return _eval(expr, inplace=inplace, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/eval.py"", line 354, in eval
    parsed_expr = Expr(expr, engine=engine, parser=parser, env=env)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 813, in __init__
    self.terms = self.parse()
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 832, in parse
    return self._visitor.visit(self.expr)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 421, in visit_Module
    return self.visit(expr, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 424, in visit_Expr
    return self.visit(node.value, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 723, in visit_Compare
    return self.visit(binop)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 536, in visit_BinOp
    op, op_class, left, right = self._maybe_transform_eq_ne(node)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 458, in _maybe_transform_eq_ne
    right = self.visit(node.right, side=""right"")
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 645, in visit_Attribute
    resolved = self.visit(value).value
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 415, in visit
    return visitor(node, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/expr.py"", line 549, in visit_Name
    return self.term_type(node.id, self.env, **kwargs)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/ops.py"", line 85, in __init__
    self._value = self._resolve_name()
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/ops.py"", line 109, in _resolve_name
    res = self.env.resolve(local_name, is_local=is_local)
  File ""/home/petr/.local/venv/lib64/python3.10/site-packages/pandas/core/computation/scope.py"", line 211, in resolve
    raise UndefinedVariableError(key, is_local) from err
pandas.errors.UndefinedVariableError: name 'A' is not defined
```

</details>

I think it is after this commit:
https://github.com/pandas-dev/pandas/commit/8b36142a0e7c1071e9fd9ff37f579ebdef056838

I can get expected behavior if I comment this condition:
https://github.com/pandas-dev/pandas/blob/main/pandas/core/computation/ops.py#L105

### Expected Behavior

```python
>>> import pandas as pd
>>> 
>>> df = pd.DataFrame({'x': [1, 2, 3], 'y': [4, 5, 6]})
>>> 
>>> class A:
...     a = 1
... 
>>> df.query('x == @A.a')  # <-- this produces the error
   x  y
0  1  4
```


### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit           : 87cfe4e38bafe7300a6003a1d18bd80f3f77c763
python           : 3.10.6.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.19.9-200.fc36.x86_64
Version          : #1 SMP PREEMPT_DYNAMIC Thu Sep 15 09:49:52 UTC 2022
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.5.0
numpy            : 1.23.3
pytz             : 2022.2.1
dateutil         : 2.8.2
setuptools       : 65.3.0
pip              : 22.2.2
Cython           : None
pytest           : 7.1.3
hypothesis       : None
sphinx           : 5.1.1
blosc            : None
feather          : None
xlsxwriter       : 3.0.3
lxml.etree       : 4.9.1
html5lib         : 1.1
pymysql          : None
psycopg2         : 2.9.3
jinja2           : 3.1.2
IPython          : 8.5.0
pandas_datareader: None
bs4              : 4.11.1
bottleneck       : 1.3.5
brotli           : None
fastparquet      : None
fsspec           : None
gcsfs            : None
matplotlib       : 3.6.0
numba            : None
numexpr          : 2.8.3
odfpy            : None
openpyxl         : 3.0.10
pandas_gbq       : None
pyarrow          : None
pyreadstat       : None
pyxlsb           : None
s3fs             : None
scipy            : 1.9.1
snappy           : None
sqlalchemy       : 1.4.41
tables           : None
tabulate         : 0.8.10
xarray           : None
xlrd             : 2.0.1
xlwt             : None
zstandard        : None
tzdata           : None

</details>
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48694/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48694/timeline,,
22,https://api.github.com/repos/pandas-dev/pandas/issues/48693,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48693/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48693/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48693/events,https://github.com/pandas-dev/pandas/pull/48693,1381404334,PR_kwDOAA0YD84_YDce,48693,ENH: Make deprecate_nonkeyword_arguments alter function signature,"{'login': 'hauntsaninja', 'id': 12621235, 'node_id': 'MDQ6VXNlcjEyNjIxMjM1', 'avatar_url': 'https://avatars.githubusercontent.com/u/12621235?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/hauntsaninja', 'html_url': 'https://github.com/hauntsaninja', 'followers_url': 'https://api.github.com/users/hauntsaninja/followers', 'following_url': 'https://api.github.com/users/hauntsaninja/following{/other_user}', 'gists_url': 'https://api.github.com/users/hauntsaninja/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/hauntsaninja/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/hauntsaninja/subscriptions', 'organizations_url': 'https://api.github.com/users/hauntsaninja/orgs', 'repos_url': 'https://api.github.com/users/hauntsaninja/repos', 'events_url': 'https://api.github.com/users/hauntsaninja/events{/privacy}', 'received_events_url': 'https://api.github.com/users/hauntsaninja/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76812, 'node_id': 'MDU6TGFiZWw3NjgxMg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Enhancement', 'name': 'Enhancement', 'color': '4E9A06', 'default': False, 'description': None}, {'id': 87485152, 'node_id': 'MDU6TGFiZWw4NzQ4NTE1Mg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Deprecate', 'name': 'Deprecate', 'color': '5319e7', 'default': False, 'description': 'Functionality to remove in pandas'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",2,2022-09-21T19:36:10Z,2022-09-22T20:12:55Z,,CONTRIBUTOR,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48693', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48693', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48693.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48693.patch', 'merged_at': None}","- [x] closes #48692
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48693/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48693/timeline,,
23,https://api.github.com/repos/pandas-dev/pandas/issues/48692,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48692/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48692/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48692/events,https://github.com/pandas-dev/pandas/issues/48692,1381401415,I_kwDOAA0YD85SVoNH,48692,ENH: Make `deprecate_nonkeyword_arguments` alter function signature,"{'login': 'hauntsaninja', 'id': 12621235, 'node_id': 'MDQ6VXNlcjEyNjIxMjM1', 'avatar_url': 'https://avatars.githubusercontent.com/u/12621235?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/hauntsaninja', 'html_url': 'https://github.com/hauntsaninja', 'followers_url': 'https://api.github.com/users/hauntsaninja/followers', 'following_url': 'https://api.github.com/users/hauntsaninja/following{/other_user}', 'gists_url': 'https://api.github.com/users/hauntsaninja/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/hauntsaninja/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/hauntsaninja/subscriptions', 'organizations_url': 'https://api.github.com/users/hauntsaninja/orgs', 'repos_url': 'https://api.github.com/users/hauntsaninja/repos', 'events_url': 'https://api.github.com/users/hauntsaninja/events{/privacy}', 'received_events_url': 'https://api.github.com/users/hauntsaninja/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76812, 'node_id': 'MDU6TGFiZWw3NjgxMg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Enhancement', 'name': 'Enhancement', 'color': '4E9A06', 'default': False, 'description': None}, {'id': 87485152, 'node_id': 'MDU6TGFiZWw4NzQ4NTE1Mg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Deprecate', 'name': 'Deprecate', 'color': '5319e7', 'default': False, 'description': 'Functionality to remove in pandas'}]",open,False,,[],,1,2022-09-21T19:33:08Z,2022-09-22T23:21:29Z,,CONTRIBUTOR,,,,"### Feature Type

- [ ] Adding new functionality to pandas

- [X] Changing existing functionality in pandas

- [ ] Removing existing functionality in pandas


### Problem Description

Currently `deprecate_nonkeyword_arguments` does not alter the function signature. This means that `help` and `??` will show the deprecated signatures. This also affects recent attempts to achieve better typing of pandas using stubtest, e.g. see https://github.com/pandas-dev/pandas/pull/47817#pullrequestreview-1047317591 and https://github.com/python/mypy/issues/13703

### Feature Description

Set the `__signature__` attr on the decorated function

### Alternative Solutions

Wait for all the removals to take place

### Additional Context

_No response_","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48692/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48692/timeline,,
24,https://api.github.com/repos/pandas-dev/pandas/issues/48691,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48691/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48691/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48691/events,https://github.com/pandas-dev/pandas/pull/48691,1381395348,PR_kwDOAA0YD84_YBiz,48691,"BUG: Restrict Timestamp(nanosecond=...) to [0, 999]","{'login': 'mroeschke', 'id': 10647082, 'node_id': 'MDQ6VXNlcjEwNjQ3MDgy', 'avatar_url': 'https://avatars.githubusercontent.com/u/10647082?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/mroeschke', 'html_url': 'https://github.com/mroeschke', 'followers_url': 'https://api.github.com/users/mroeschke/followers', 'following_url': 'https://api.github.com/users/mroeschke/following{/other_user}', 'gists_url': 'https://api.github.com/users/mroeschke/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/mroeschke/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/mroeschke/subscriptions', 'organizations_url': 'https://api.github.com/users/mroeschke/orgs', 'repos_url': 'https://api.github.com/users/mroeschke/repos', 'events_url': 'https://api.github.com/users/mroeschke/events{/privacy}', 'received_events_url': 'https://api.github.com/users/mroeschke/received_events', 'type': 'User', 'site_admin': False}","[{'id': 87485152, 'node_id': 'MDU6TGFiZWw4NzQ4NTE1Mg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Deprecate', 'name': 'Deprecate', 'color': '5319e7', 'default': False, 'description': 'Functionality to remove in pandas'}, {'id': 3877216987, 'node_id': 'LA_kwDOAA0YD87nGaLb', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timestamp', 'name': 'Timestamp', 'color': '5DE3E8', 'default': False, 'description': 'pd.Timestamp and associated methods'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/97', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/97', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/97/labels', 'id': 8341565, 'node_id': 'MI_kwDOAA0YD84Af0g9', 'number': 97, 'title': '1.6', 'description': '', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 7, 'closed_issues': 132, 'state': 'open', 'created_at': '2022-08-23T17:53:13Z', 'updated_at': '2022-09-23T12:27:51Z', 'due_on': None, 'closed_at': None}",0,2022-09-21T19:26:47Z,2022-09-22T20:11:09Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48691', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48691', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48691.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48691.patch', 'merged_at': None}","- [x] closes #48538 (Replace xxxx with the Github issue number)
- [x] closes #48255 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added an entry in the latest `doc/source/whatsnew/v1.6.0.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48691/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48691/timeline,,
25,https://api.github.com/repos/pandas-dev/pandas/issues/48689,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48689/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48689/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48689/events,https://github.com/pandas-dev/pandas/pull/48689,1381296434,PR_kwDOAA0YD84_XsLw,48689,"DEPR: enforce Timedelta freq, delta, is_populated deprecations","{'login': 'jbrockmendel', 'id': 8078968, 'node_id': 'MDQ6VXNlcjgwNzg5Njg=', 'avatar_url': 'https://avatars.githubusercontent.com/u/8078968?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jbrockmendel', 'html_url': 'https://github.com/jbrockmendel', 'followers_url': 'https://api.github.com/users/jbrockmendel/followers', 'following_url': 'https://api.github.com/users/jbrockmendel/following{/other_user}', 'gists_url': 'https://api.github.com/users/jbrockmendel/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jbrockmendel/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jbrockmendel/subscriptions', 'organizations_url': 'https://api.github.com/users/jbrockmendel/orgs', 'repos_url': 'https://api.github.com/users/jbrockmendel/repos', 'events_url': 'https://api.github.com/users/jbrockmendel/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jbrockmendel/received_events', 'type': 'User', 'site_admin': False}","[{'id': 49597148, 'node_id': 'MDU6TGFiZWw0OTU5NzE0OA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timedelta', 'name': 'Timedelta', 'color': '5319e7', 'default': False, 'description': 'Timedelta data type'}, {'id': 87485152, 'node_id': 'MDU6TGFiZWw4NzQ4NTE1Mg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Deprecate', 'name': 'Deprecate', 'color': '5319e7', 'default': False, 'description': 'Functionality to remove in pandas'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/42', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/42', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/42/labels', 'id': 1945020, 'node_id': 'MDk6TWlsZXN0b25lMTk0NTAyMA==', 'number': 42, 'title': '2.0', 'description': '', 'creator': {'login': 'jreback', 'id': 953992, 'node_id': 'MDQ6VXNlcjk1Mzk5Mg==', 'avatar_url': 'https://avatars.githubusercontent.com/u/953992?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jreback', 'html_url': 'https://github.com/jreback', 'followers_url': 'https://api.github.com/users/jreback/followers', 'following_url': 'https://api.github.com/users/jreback/following{/other_user}', 'gists_url': 'https://api.github.com/users/jreback/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jreback/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jreback/subscriptions', 'organizations_url': 'https://api.github.com/users/jreback/orgs', 'repos_url': 'https://api.github.com/users/jreback/repos', 'events_url': 'https://api.github.com/users/jreback/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jreback/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 6, 'state': 'open', 'created_at': '2016-08-16T13:21:24Z', 'updated_at': '2022-09-21T18:08:29Z', 'due_on': '2022-12-31T08:00:00Z', 'closed_at': None}",0,2022-09-21T17:57:50Z,2022-09-22T14:50:29Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48689', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48689', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48689.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48689.patch', 'merged_at': None}","- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48689/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48689/timeline,,
26,https://api.github.com/repos/pandas-dev/pandas/issues/48688,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48688/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48688/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48688/events,https://github.com/pandas-dev/pandas/issues/48688,1381275478,I_kwDOAA0YD85SVJdW,48688,BUG: explicit tz=None ignored in Timestamp constructor,"{'login': 'jbrockmendel', 'id': 8078968, 'node_id': 'MDQ6VXNlcjgwNzg5Njg=', 'avatar_url': 'https://avatars.githubusercontent.com/u/8078968?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jbrockmendel', 'html_url': 'https://github.com/jbrockmendel', 'followers_url': 'https://api.github.com/users/jbrockmendel/followers', 'following_url': 'https://api.github.com/users/jbrockmendel/following{/other_user}', 'gists_url': 'https://api.github.com/users/jbrockmendel/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jbrockmendel/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jbrockmendel/subscriptions', 'organizations_url': 'https://api.github.com/users/jbrockmendel/orgs', 'repos_url': 'https://api.github.com/users/jbrockmendel/repos', 'events_url': 'https://api.github.com/users/jbrockmendel/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jbrockmendel/received_events', 'type': 'User', 'site_admin': False}","[{'id': 76811, 'node_id': 'MDU6TGFiZWw3NjgxMQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Bug', 'name': 'Bug', 'color': 'e10c02', 'default': False, 'description': None}, {'id': 211840, 'node_id': 'MDU6TGFiZWwyMTE4NDA=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timeseries', 'name': 'Timeseries', 'color': 'AFEEEE', 'default': False, 'description': None}, {'id': 60458168, 'node_id': 'MDU6TGFiZWw2MDQ1ODE2OA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timezones', 'name': 'Timezones', 'color': '5319e7', 'default': False, 'description': 'Timezone data dtype'}]",open,False,,[],,0,2022-09-21T17:37:56Z,2022-09-21T21:27:09Z,,MEMBER,,,,"```python
In [14]: Timestamp(datetime(2022, 1, 1, tzinfo=timezone.utc), tz=None)
Out[14]: Timestamp('2022-01-01 00:00:00+0000', tz='UTC')
```

If a user explicitly passes tz=None and a tzaware string or datetime object, we should raise.  This was fixed for DatetimeIndex in GH#48659
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48688/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48688/timeline,,
27,https://api.github.com/repos/pandas-dev/pandas/issues/48687,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48687/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48687/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48687/events,https://github.com/pandas-dev/pandas/pull/48687,1381248223,PR_kwDOAA0YD84_Xh9T,48687,DOC: Clarify exceptions that are internal only,"{'login': 'mroeschke', 'id': 10647082, 'node_id': 'MDQ6VXNlcjEwNjQ3MDgy', 'avatar_url': 'https://avatars.githubusercontent.com/u/10647082?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/mroeschke', 'html_url': 'https://github.com/mroeschke', 'followers_url': 'https://api.github.com/users/mroeschke/followers', 'following_url': 'https://api.github.com/users/mroeschke/following{/other_user}', 'gists_url': 'https://api.github.com/users/mroeschke/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/mroeschke/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/mroeschke/subscriptions', 'organizations_url': 'https://api.github.com/users/mroeschke/orgs', 'repos_url': 'https://api.github.com/users/mroeschke/repos', 'events_url': 'https://api.github.com/users/mroeschke/events{/privacy}', 'received_events_url': 'https://api.github.com/users/mroeschke/received_events', 'type': 'User', 'site_admin': False}","[{'id': 134699, 'node_id': 'MDU6TGFiZWwxMzQ2OTk=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Docs', 'name': 'Docs', 'color': '3465A4', 'default': False, 'description': None}, {'id': 42670965, 'node_id': 'MDU6TGFiZWw0MjY3MDk2NQ==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Error%20Reporting', 'name': 'Error Reporting', 'color': 'ffa0ff', 'default': False, 'description': 'Incorrect or improved errors from pandas'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/97', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/97', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/97/labels', 'id': 8341565, 'node_id': 'MI_kwDOAA0YD84Af0g9', 'number': 97, 'title': '1.6', 'description': '', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 7, 'closed_issues': 132, 'state': 'open', 'created_at': '2022-08-23T17:53:13Z', 'updated_at': '2022-09-23T12:27:51Z', 'due_on': None, 'closed_at': None}",0,2022-09-21T17:12:13Z,2022-09-22T16:23:07Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48687', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48687', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48687.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48687.patch', 'merged_at': None}","xref https://github.com/pandas-dev/pandas/pull/48088#issuecomment-1252924401
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48687/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48687/timeline,,
28,https://api.github.com/repos/pandas-dev/pandas/issues/48686,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48686/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48686/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48686/events,https://github.com/pandas-dev/pandas/pull/48686,1381235570,PR_kwDOAA0YD84_XfO-,48686,"BUG: to_datetime(tz_mix, utc=True) converts to UTC","{'login': 'mroeschke', 'id': 10647082, 'node_id': 'MDQ6VXNlcjEwNjQ3MDgy', 'avatar_url': 'https://avatars.githubusercontent.com/u/10647082?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/mroeschke', 'html_url': 'https://github.com/mroeschke', 'followers_url': 'https://api.github.com/users/mroeschke/followers', 'following_url': 'https://api.github.com/users/mroeschke/following{/other_user}', 'gists_url': 'https://api.github.com/users/mroeschke/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/mroeschke/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/mroeschke/subscriptions', 'organizations_url': 'https://api.github.com/users/mroeschke/orgs', 'repos_url': 'https://api.github.com/users/mroeschke/repos', 'events_url': 'https://api.github.com/users/mroeschke/events{/privacy}', 'received_events_url': 'https://api.github.com/users/mroeschke/received_events', 'type': 'User', 'site_admin': False}","[{'id': 211840, 'node_id': 'MDU6TGFiZWwyMTE4NDA=', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timeseries', 'name': 'Timeseries', 'color': 'AFEEEE', 'default': False, 'description': None}, {'id': 60458168, 'node_id': 'MDU6TGFiZWw2MDQ1ODE2OA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Timezones', 'name': 'Timezones', 'color': '5319e7', 'default': False, 'description': 'Timezone data dtype'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/98', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/98/labels', 'id': 8438761, 'node_id': 'MI_kwDOAA0YD84AgMPp', 'number': 98, 'title': '1.5.1', 'description': 'on-merge: backport to 1.5.x', 'creator': {'login': 'datapythonista', 'id': 10058240, 'node_id': 'MDQ6VXNlcjEwMDU4MjQw', 'avatar_url': 'https://avatars.githubusercontent.com/u/10058240?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/datapythonista', 'html_url': 'https://github.com/datapythonista', 'followers_url': 'https://api.github.com/users/datapythonista/followers', 'following_url': 'https://api.github.com/users/datapythonista/following{/other_user}', 'gists_url': 'https://api.github.com/users/datapythonista/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/datapythonista/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/datapythonista/subscriptions', 'organizations_url': 'https://api.github.com/users/datapythonista/orgs', 'repos_url': 'https://api.github.com/users/datapythonista/repos', 'events_url': 'https://api.github.com/users/datapythonista/events{/privacy}', 'received_events_url': 'https://api.github.com/users/datapythonista/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 17, 'state': 'open', 'created_at': '2022-09-19T20:12:38Z', 'updated_at': '2022-09-23T13:45:39Z', 'due_on': None, 'closed_at': None}",0,2022-09-21T17:00:55Z,2022-09-22T20:02:24Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48686', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48686', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48686.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48686.patch', 'merged_at': None}","- [x] closes #48678 (Replace xxxx with the Github issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added an entry in the latest `doc/source/whatsnew/v1.5.1.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48686/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48686/timeline,,
29,https://api.github.com/repos/pandas-dev/pandas/issues/48685,https://api.github.com/repos/pandas-dev/pandas,https://api.github.com/repos/pandas-dev/pandas/issues/48685/labels{/name},https://api.github.com/repos/pandas-dev/pandas/issues/48685/comments,https://api.github.com/repos/pandas-dev/pandas/issues/48685/events,https://github.com/pandas-dev/pandas/pull/48685,1381219072,PR_kwDOAA0YD84_XbvD,48685,DEPR: enforce deprecations on DateOffset methods,"{'login': 'jbrockmendel', 'id': 8078968, 'node_id': 'MDQ6VXNlcjgwNzg5Njg=', 'avatar_url': 'https://avatars.githubusercontent.com/u/8078968?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jbrockmendel', 'html_url': 'https://github.com/jbrockmendel', 'followers_url': 'https://api.github.com/users/jbrockmendel/followers', 'following_url': 'https://api.github.com/users/jbrockmendel/following{/other_user}', 'gists_url': 'https://api.github.com/users/jbrockmendel/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jbrockmendel/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jbrockmendel/subscriptions', 'organizations_url': 'https://api.github.com/users/jbrockmendel/orgs', 'repos_url': 'https://api.github.com/users/jbrockmendel/repos', 'events_url': 'https://api.github.com/users/jbrockmendel/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jbrockmendel/received_events', 'type': 'User', 'site_admin': False}","[{'id': 53181044, 'node_id': 'MDU6TGFiZWw1MzE4MTA0NA==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Frequency', 'name': 'Frequency', 'color': '0052cc', 'default': False, 'description': 'DateOffsets'}, {'id': 87485152, 'node_id': 'MDU6TGFiZWw4NzQ4NTE1Mg==', 'url': 'https://api.github.com/repos/pandas-dev/pandas/labels/Deprecate', 'name': 'Deprecate', 'color': '5319e7', 'default': False, 'description': 'Functionality to remove in pandas'}]",open,False,,[],"{'url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/42', 'html_url': 'https://github.com/pandas-dev/pandas/milestone/42', 'labels_url': 'https://api.github.com/repos/pandas-dev/pandas/milestones/42/labels', 'id': 1945020, 'node_id': 'MDk6TWlsZXN0b25lMTk0NTAyMA==', 'number': 42, 'title': '2.0', 'description': '', 'creator': {'login': 'jreback', 'id': 953992, 'node_id': 'MDQ6VXNlcjk1Mzk5Mg==', 'avatar_url': 'https://avatars.githubusercontent.com/u/953992?v=4', 'gravatar_id': '', 'url': 'https://api.github.com/users/jreback', 'html_url': 'https://github.com/jreback', 'followers_url': 'https://api.github.com/users/jreback/followers', 'following_url': 'https://api.github.com/users/jreback/following{/other_user}', 'gists_url': 'https://api.github.com/users/jreback/gists{/gist_id}', 'starred_url': 'https://api.github.com/users/jreback/starred{/owner}{/repo}', 'subscriptions_url': 'https://api.github.com/users/jreback/subscriptions', 'organizations_url': 'https://api.github.com/users/jreback/orgs', 'repos_url': 'https://api.github.com/users/jreback/repos', 'events_url': 'https://api.github.com/users/jreback/events{/privacy}', 'received_events_url': 'https://api.github.com/users/jreback/received_events', 'type': 'User', 'site_admin': False}, 'open_issues': 27, 'closed_issues': 6, 'state': 'open', 'created_at': '2016-08-16T13:21:24Z', 'updated_at': '2022-09-21T18:08:29Z', 'due_on': '2022-12-31T08:00:00Z', 'closed_at': None}",0,2022-09-21T16:45:31Z,2022-09-22T14:53:15Z,,MEMBER,,False,"{'url': 'https://api.github.com/repos/pandas-dev/pandas/pulls/48685', 'html_url': 'https://github.com/pandas-dev/pandas/pull/48685', 'diff_url': 'https://github.com/pandas-dev/pandas/pull/48685.diff', 'patch_url': 'https://github.com/pandas-dev/pandas/pull/48685.patch', 'merged_at': None}","- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.
","{'url': 'https://api.github.com/repos/pandas-dev/pandas/issues/48685/reactions', 'total_count': 0, '+1': 0, '-1': 0, 'laugh': 0, 'hooray': 0, 'confused': 0, 'heart': 0, 'rocket': 0, 'eyes': 0}",https://api.github.com/repos/pandas-dev/pandas/issues/48685/timeline,,
